src/crud/useFetchMediaUrl.ts

```diff
@@ -130,7 +130,7 @@ export const useFetchMediaUrl = (reporter?: any) => {
               if (cancelled()) return;
               dispatch({ payload: safeUrl, type: MediaSt.FETCHED });
               return;
-            } else if (!props.current.auth) {
+            } else if (!props.current.auth || !props.current.auth.accessToken) {
```

Comment:
you could probably simplify the test as !props.current.auth?.accessToken (and not need the ||)